.github/workflows/ci-quality.yml

```
@@ -0,0 +1,201 @@
+              echo "$MISSING" | sed 's/^/    /'
+              FAILED=1
+            fi
+            if [ -n "$EXTRA" ]; then
+              echo "✗ ${B}.json ha chiavi extra non presenti in ${A}.json:"
+              echo "$EXTRA" | sed 's/^/    /'
+              FAILED=1
+            fi
+          done
+          # 2. Placeholder parity: %s / %d must match count between en_US and de_DE
+          python3 - <<'PYEOF'
+import json, re, sys
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)